editor/components/RelatedRecordTable.tsx

```diff
@@ -106,6 +110,34 @@ export interface RowActionComponentProps<
   isEditingColumn?: boolean | null;
 }
 
+interface SortSettings<T extends Record<string, unknown>> {
+  col: null | ColDataCardDef<T>;
+  asc: boolean;
+}
+
+const useSortedRows = <T extends Record<string, unknown>>({
+  records,
+  sortSettings,
+}: {
+  records: T[];
+  sortSettings: SortSettings<T>;
+}) =>
+  useMemo(() => {
+    const sortCol = sortSettings.col;
+    if (!sortCol) {
+      return records;
+    }
+    const compareFunc = sortCol.compareFunc
+      ? sortCol.compareFunc
+      : compareStrings;
```

Comment:
I don't feel strongly about it, but not having a default/fallback comparator might be a way to signal to devs that they need to "fill in the blank" in the config since this default sort could appear to work on other data types. This change would probably require explicitly setting `compareStrings` for all the string in the column config too though so idk.

Reply:
Yeah, there's definitely a gotcha built in by handling everything as strings. We could get fancy and have a little hook that checks the data type of the first row to determine the default sort func, so that it sorts numbers/bools differently. TBH i would kinda prefer to do that than add `compareStrings` to all the configs 🤔

Reply:
ah, and because of our dual-duty `ColDataCardDef` type, this would also require us to add a `compareFunc` def to columns which do server-side sorting in our list tables 😖